src/main/java/com/comphenix/protocol/injector/packet/PacketRegistry.java

```diff
@@ -461,7 +461,10 @@ private static synchronized Register createRegisterV1_20_5() {
 						}
 
 						Object serializer = idCodecEntrySerializerAccessor.invoke(entry);
-						result.classToCodec.put(packetClass, new WrappedStreamCodec(serializer));
+						Field outerThisField = serializer.getClass().getDeclaredField("this$0");
```

Comment:
Should we use FuzzyReflection here in case some JVM implements Lambdas differently?